ldsw/.htaccess

```diff
@@ -0,0 +1,5 @@
+Header set Access-Control-Allow-Origin *
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^.*$ https://raw.githubusercontent.com/s2350726/ldsw/main/university.ttl [R=303,L]
+RewriteRule ^.*$ https://raw.githubusercontent.com/s2350726/ldsw/main/university.owl [R=303,NE,L]
```

Comment:
The first rule is going to match everything.  You may want to setup content negotiation.  (See other files for examples.)